modcc/printer/gpuprinter.cpp

```diff
@@ -429,8 +429,8 @@ void emit_api_body_cu(std::ostream& out, APIMethod* e, bool is_point_proc, bool
                 out << "unsigned lane_mask_ = arb::gpu::ballot(0xffffffff, tid_<n_);\n";
             }
         }
-        ppack && out << "PPACK_IFACE_BLOCK;\n";
-        cv_loop && out << "if (tid_<n_) {\n" << indent;
+        flags.ppack_iface && out << "PPACK_IFACE_BLOCK;\n";
+        flags.cv_loop && out << "if (tid_<n_) {\n" << indent;
```

Comment:
More readable, and inline with your changes in `cprinter`:
```suggestion
        if (flags.ppack_iface) out << "PPACK_IFACE_BLOCK;\n";
        if (flags.cv_loop) out << "if (tid_<n_) {\n" << indent;
```